src/script/vco-ie/main_warhammer/wh2_dlc09_tmb_followers_of_nagash/missions.lua

```diff
@@ -0,0 +1,162 @@
+local missions = {
+	[[
+ 		mission
+		{
+			victory_type vco_victory_type_tmb_ark_1_thy_kingdom_come;
+			key wh_main_short_victory;
+			issuer CLAN_ELDERS;
+			primary_objectives_and_payload
+			{
+				objective
+				{
+					override_text mission_text_text_vco_tmb_ark_1_thy_kingdom_come;
+					type SCRIPTED;
+					script_key vco_dummy;
+				}
+
+				objective
+				{
+
+					type DESTROY_FACTION;
+
+					faction wh2_dlc09_tmb_khemri;
+
+					confederation_valid;
+				}
+
+				objective
+				{
+					type CONTROL_N_PROVINCES_INCLUDING;
```

Comment:
We got the problem with CONTROL_N_PROVINCES_FROM here. It has to be addressed

Reply:
I would just bump the number up to 7 and call it a day.

Reply:
Mannfred has the same objective as well. I would also change his to 7.